applications/ws-affiliation-rnsr/Dockerfile

```
@@ -1,29 +1,23 @@
 # syntax=docker/dockerfile:1.2
 FROM inistcnrs/lodex-workers-python:4.0.11 as build1
-RUN --mount=type=secret,id=webdav_login \
-    --mount=type=secret,id=webdav_password \
-    --mount=type=secret,id=webdav_url \
+RUN --mount=type=secret,id=webdav_login cat /secrets/login \
```

Dans les lignes `mount`, pas besoin de `cat` (`cat` est une commande en ligne de bash, qui permet de concaténer le contenu de fichiers, et comme qui peut le plus peut le moins, il peut aussi afficher le contenu d'un fichier: il le met sur la sortie standard `stdout`; on s'en servira plus bas).

```suggestion
RUN --mount=type=secret,id=webdav_login
```